manifest_staging/charts/csi-secrets-store-provider-azure/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "sscdpa.fullname" . }}
```

Comment:
If the release name is used for psp and the release name begins with `z-*` for instance, then the pods will get blocked by the restricted policy if it exists. Should the policy instead have a custom name?

cc @sozercan